client/me/get-apps/mobile-download-card.jsx

```diff
@@ -6,46 +6,270 @@
+
+		if ( element !== undefined ) {
+			return element.numeric_code;
+		}
+
+		return null;
+	}
+
+	userSettingsHaveBeenLoadedWithAccountRecoveryPhone() {
+		return this.props.hasUserSettings && this.props.hasLoadedAccountRecoveryPhone;
+	}
+
+	render() {
+		const { translate } = this.props;
+
+		const hasAllData = this.userSettingsHaveBeenLoadedWithAccountRecoveryPhone();
+		const { countryCode, number, isValid } = this.getPreferredNumber();
+		const { isMobile } = userAgent;
+		const featureIsEnabled = config.isEnabled( 'get-apps-sms' ) && ! isMobile;
+
+		return (
+			<Card className="get-apps__mobile">
+				<div className="get-apps__store-subpanel">
+					<div className="get-apps__card-text">
+						<h3 className="get-apps__card-title">{ translate( 'Mobile Apps' ) }</h3>
+						<p className="get-apps__description">
+							{ translate( 'WordPress at your fingertips.' ) }
+						</p>
+					</div>
+					<div className="get-apps__badges">
+						<AppsBadge
+							storeLink="https://play.google.com/store/apps/details?id=org.wordpress.android&referrer=utm_source%3Dcalypso-get-apps%26utm_medium%3Dweb%26utm_campaign%3Dmobile-download-promo-pages"
+							storeName={ 'android' }
+							titleText={ translate( 'Download the WordPress Android mobile app.' ) }
+							altText={ translate( 'Google Play Store download badge' ) }
+						/>
+						<AppsBadge
+							storeLink="https://itunes.apple.com/app/apple-store/id335703880?pt=299112&ct=calpyso-get-apps-button&mt=8"
+							storeName={ 'ios' }
+							titleText={ translate( 'Download the WordPress iOS mobile app.' ) }
+							altText={ translate( 'Apple App Store download badge' ) }
+						/>
+					</div>
+				</div>
+
+				{ featureIsEnabled ? (
+					<div className="get-apps__sms-subpanel">
+						<p>
+							<strong>{ translate( 'Ready to WordPress on the go?' ) }</strong>
+							<br />
+							{ translate(
+								'We’ll send you an SMS message with a download link for the right app for your mobile device.'
+							) }
+						</p>
+
+						<div className="get-apps__sms-field-wrapper">
+							<QuerySmsCountries />
+							<QueryAccountRecoverySettings />
+							<QueryUserSettings />
+
+							{ hasAllData ? (
+								<FormPhoneInput
+									countriesList={ this.props.countriesList }
+									initialCountryCode={ countryCode }
+									initialPhoneNumber={ number }
+									phoneInputProps={ {
+										onKeyUp: this.onKeyUp,
+									} }
+									onChange={ this.onChange }
+								/>
+							) : (
+								<>
+									<FormPhoneInput countriesList={ this.props.countriesList } isDisabled={ true } />
+								</>
+							) }
+						</div>
+						<div className="get-apps__sms-button-wrapper">
+							<p>{ translate( 'Standard SMS rates may apply' ) }</p>
+
+							<Button
+								className="get-apps__sms-button"
+								onClick={ this.onSubmit }
+								disabled={ ! isValid }
+							>
+								{ translate( 'Text me a link' ) }
+							</Button>
+						</div>
+					</div>
+				) : (
```

Comment:
You could drop that part if you write it this way:
```
{ featureIsEnabled &&
    <div>
    ....
    </div>
}
```

Reply:
I didn't know that was possible! Thanks – updated in c015d7bfe1643107bcc79776349b9a3810b92746